qiskit/primitives/containers/data_bin.py

```diff
@@ -93,9 +93,6 @@ def __init__(self, *, shape: ShapeInput = (), **data):
     def __len__(self):
         return len(self._data)
 
-    def __setattr__(self, *_):
-        raise NotImplementedError
-
```

Comment:
Is it ok to remove this? This `__setattr__` is preventing the serialization of the job result

Reply:
Could you answer the question, @ihincks?

Reply:
#12787 asked the same question.

Reply:
Please go ahead and remove it. Apologies, this completely fell off my radar.